### ISSUE TITLE
Failure to synchronize with native YouTube playback speed control

### ISSUE THREAD

**Issue by Xeinok-Pals** [igrigorik/videospeed#658]
While using this extension, YouTube (Settings cogwheel > Playback speed) is not adjusted to what the extension has changed. The video itself plays at the speed you adjust using the extension, but this causes issues in the closed captions de-synchronizing until you also manually adjust the YouTube playback speed setting to match.

![image](https://user-images.githubusercontent.com/65511279/82159853-360a5180-9846-11ea-8895-b1adcb26e6ac.png)

**Comment by ChadBailey** [comment]
thanks for the report, i think this may be an unintentional side-effect from a fix i put in a while back. you can think of it like i basically made the website unaware of the adjustments taking place from the controller while allowing the controller to always be aware of the adjustments the website was making. this was done to fix some problematic sites which would forcefully reset the speed each time it was adjusted.

i think the solution may be to move back to a model where the event blocking is only occurring on the problematic sites. i had hoped blocking it as a global policy would not have any ill-effects, but seems this is one of many possible side-effects.

tl;dr - my bad, i broke it. i'll see if i can get a fix in

**Comment by Xeinok-Pals** [comment]
No problem, thanks for all your hard work on this very useful extension
